std/typecons.d

```diff
@@ -792,10 +852,36 @@ template Tuple(Specs...)
          * Returns:
          *     The string representation of this `Tuple`.
          */
-        string toString()()
+        string toString()() const
         {
-            import std.conv : to;
-            return this.to!string;
+            import std.array : appender;
+            auto app = appender!string();
+            this.toString((const(char)[] chunk) => app ~= chunk);
+            return app.data;
+        }
+
+        ///
+        unittest
+        {
+            import std.format    : format, FormatException;
+            import std.format : format;
```

Comment:
It's good to be mentioned. If it really is a performance issue, it has to be done. It shouldn't be a big thing, or was it in `std.container`?